components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapterFactory.java

```diff
@@ -61,7 +62,20 @@ public String getType() {
 
     @Override
     public String getUsageTips() {
-        return resourceBundle.getString(UIEventAdapterConstants.ADAPTER_USAGE_TIPS_UI);
+        int defaultHttpPort = 9763;
+        int defaultHttpsPort = 9443;
+        int httpPort;
+        int httpsPort;
+        int portOffset = getPortOffset();
+
+        if (portOffset == 0) {
+            httpPort = defaultHttpPort;
+            httpsPort = defaultHttpsPort;
+        } else {
+            httpPort = defaultHttpPort + portOffset;
+            httpsPort = defaultHttpsPort + portOffset;
+        }
```

Comment:
@thiliA 

Shall we set the default ports as a constant and set the port offset value at initialization (since port offset won't change during server runtime). Otherwise these values will be calculated each time getUsageTips() method is called.